crates/ruma-events/src/notification_attribute_data.rs

```diff
@@ -24,6 +24,14 @@ pub struct NotificationAttributeDataEventContent {
     pub mentions: Mentions,
 }
 
+impl NotificationAttributeDataEventContent {
+    /// Creates a new `NotificationAttributeDataEventContent with the given keywords and mentions
+    /// object.
```

Comment:
```suggestion
    /// Creates a new `NotificationAttributeDataEventContent with the given keywords and mentions.
```

I don't think "object" adds anything here.

Reply:
Modified in one of the rebased commits.